src/__tests__/App.test.tsx

```diff
@@ -88,7 +87,6 @@ describe('App', () => {
           'Error when requesting treeherder: (500) Internal Server Error',
         ),
       );
-      expect(console.error).toHaveBeenCalledTimes(14);
```

Comment:
I find these console.error checks to be brittle (because the errors are output in several places, with Await, React, etc) and decided to remove them here and below. I kept the ones that are more consistent.

Reply:
They really are. Thanks for removing them.